assets/javascripts/discourse/components/chat-channel-about-view.js

```diff
@@ -19,8 +19,13 @@ export default class ChatChannelAboutView extends Component {
 
     return ChatApi.followChatChannel(this.channel.id)
       .then((membership) => {
-        this.channel.set("following", true);
-        this.channel.set("memberships_count", membership.user_count);
+        this.channel.setProperties({
```

Comment:
This is now done in a few places in the app, maybe we can move it into the `followChatChannel` function, passing the whole channel object instead of just the ID?